airbyte-webapp/README.md

```diff
@@ -8,6 +8,14 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm ci`
+
+Installs essential modules Ex. "orval"
```

Comment:
This is not what `npm ci` actually does. `npm ci` (see [docs](https://docs.npmjs.com/cli/v8/commands/npm-ci) is installing dependencies and failing more easily if there's a missmatch found instead of updating the `package-lock.json` file. I don't think we need to document `npm ci` at all, since you should never require to use this when developing on this project.